docs/src/main/sphinx/sql/drop-table.rst

```diff
@@ -15,7 +15,8 @@ Description
 Drops an existing table.
 
 The optional ``IF EXISTS`` clause causes the error to be suppressed if
-the table does not exist.
+the table does not exist. The error will not be supressed if a view with the
```

Comment:
But I am also wondering if that is true ... what kind of view are we talking about here? A trino native view or a view in the underlying database? And is this consistent across connectors..

Reply:
@hashhar can you clarify how this works for views (and what kind of views)

Reply:
Trino doesn't know if something is a view in remote database (at-least for JDBC connectors or things like BigQuery). So those views are tables for Trino. i.e. DROP TABLE will work on views that exist in remote system.

This is specifically talking about Trino views.

cc: @findinpath  for double checking me regarding other connectors like hive.

Reply:
So we most likely have to clarify what type of view we are talking about here .. since from @hashhar is saying .. this current change would be incorrect for views in JDBC connector data sources and similar. 

Also thinking about this some more I am wondering why we want to add this message at all. E.g. will it actually drop a native view if is exists with the "Drop table" command?

Reply:
Oh.. so this applies to Hive views or Trino views in your case?

Reply:
Trino views with hive connector (not sure how they are represented by Hive under the hood).

Reply:
So @findinpath and @hashhar .. how do we go from here.. I am not sure this doc change on its own is sufficient or even correct.